frontend/packages/rhoas-plugin/src/catalog/providers/useRhoasCatalog.tsx

```diff
@@ -0,0 +1,160 @@
+        </FlexItem>
+        <FlexItem>{token}</FlexItem>
+      </Flex>
+    );
+  };
+
+  const drawerDescription = (
+    <Flex direction={{ default: 'column' }}>
+      <FlexItem>
+        <TextContent>
+          <Text component={TextVariants.p}>
+            Installed cluster-side, creates resources in specific namespaces. The RHOAS operator
+            could include these services below:
+          </Text>
+          <Text component={TextVariants.p}>
+            <b>Kafka Connect:</b> We will add a descsription for Kafka Connect here.
+          </Text>
+          <Text component={TextVariants.p}>
+            <b>API Management:</b> We will add a description for API Management here.
+          </Text>
+          <Text component={TextVariants.p}>
+            <b>Red Hat Open Data Hub:</b> We will add a description for Red Hat Open Data Hub here.
+          </Text>
+        </TextContent>
+      </FlexItem>
+      <Divider component="li" />
+      <FlexItem>
+        <AccessManagedServices />
+      </FlexItem>
+    </Flex>
+  );
+
+  const detailsDescriptions = [
+    {
+      value: drawerDescription,
+    },
+  ];
+
+  const managedServicesCard: CatalogItem[] = [
+    {
+      name: 'Red Hat OpenShift Application Services',
+      type: CATALOG_TYPE,
+      uid: 'rhoas-uid',
+      description: tokenStatusFooter(),
+      provider: 'Red Hat',
+      tags: ['Kafka', 'service', 'managed'],
+      creationTimestamp: '2019-09-04T13:56:06Z',
+      documentationUrl: 'Refer Documentation',
+      attributes: {
+        version: '1',
+      },
+      icon: {
+        class: 'ManagedServicesIcon',
+        url: operatorIcon,
+      },
+      cta: {
+        label: undefined,
+        href: '',
+      },
+      details: {
+        properties: [{ label: 'Type', value: 'Red Hat Managed Service' }],
+        descriptions: detailsDescriptions,
+      },
+    },
+  ];
+
+  const managedKafkaCard: CatalogItem[] = [
+    {
+      name: 'Red Hat OpenShift Streams for Apache Kafka',
+      type: CATALOG_TYPE,
+      uid: 'rhosak-uid',
+      description: 'OpenShift Streams for Apache Kafka',
+      provider: 'Red Hat',
+      tags: ['Kafka', 'service', 'managed'],
+      creationTimestamp: '2019-09-04T13:56:06Z',
+      documentationUrl: 'Refer Documentation',
+      attributes: {
+        version: '1',
+      },
+      icon: {
+        class: 'kafkaIcon',
+        url: managedKafkaIcon,
+      },
+      cta: {
+        label: 'Connect',
+        href,
+      },
+      details: {
+        properties: [{ label: 'Type', value: 'Red Hat Managed Service' }],
+        descriptions: [
+          {
+            value: (
+              <p>
+                Here is where we provide the description for Red Hat OpenShift Streams for Apache
+                Kafka
+              </p>
+            ),
+          },
+        ],
+      },
+    },
+  ];
+
+  const services = React.useMemo(() => (tokenSecret ? managedKafkaCard : managedServicesCard), [
```

Comment:
shall we add if `tokenSecret` is loaded , exists and has no error?

Reply:
this will be coming soon when we add error handling